store/src/main/scala/net/spals/leagur/store/hyperdex/impl/HyperdexStore.scala

```diff
@@ -0,0 +1,41 @@
+package net.spals.leagur.store.hyperdex.impl
+
+import java.util.{Optional, Collections}
+
+import com.netflix.governator.annotations.AutoBindSingleton
+import net.spals.leagur.model.Entity
+import net.spals.leagur.store.Store
+
+import scala.collection.JavaConverters._
+
+/**
+ * Hyperdex implementation of {@link Store}
+ *
+ * @author tkral
+ */
+@AutoBindSingleton(classOf[Store])
+class HyperdexStore extends Store {
```

Comment:
A skeleton for the HyperDex implementation of `Store`. We'll need to fill this out, which I'd imagine isn't that hard. Also need to work on some configuration injection so we know how to contact the HyperDex coordinator.